docs/SchemeFileFormat.md

```diff
@@ -78,12 +78,13 @@ For example: `my_set/prod_ns/deny_all_policy`. If there are multiple policies na
 #### <a name="outputconfig"></a>Output Configuration object
 The supported entries in the outputConfiguration object are as follows:
 
-| Field | Description | Value |
-|-------|-------------|-------|
-|outputFormat|Output format specification.|string [ txt / yaml / csv / md / dot ] |
-|outputPath|A file path to redirect output into.|string|
-|outputEndpoints|Choose endpoints type in output.|string [ pods / deployments ]|
-|subset| A dict object with the defined subset elements to display in the output|[subset](#subset) object|
+| Field           | Description                                                                      | Value                                  |
+|-----------------|----------------------------------------------------------------------------------|----------------------------------------|
+| outputFormat    | Output format specification.                                                     | string [ txt / yaml / csv / md / dot ] |
+| outputPath      | A file path to redirect output into.                                             | string                                 |
+| outputEndpoints | Choose endpoints type in output.                                                 | string [ pods / deployments ]          |
+| subset          | A dict object with the defined subset elements to display in the output          | [subset](#subset) object               |
+| printAllPairs   | Choose if to print all counter peer pairs causing the query result in the output | bool                                   |
```

Comment:
I suggest to replace `PrintAllPairs` with something like `FullExplanation`. This will be handy for (future) queries where the explanation might look different.
Note that the explanation is usually more than just a pair of peers. It also shows protocols, etc.